src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewModel.swift

```diff
@@ -205,7 +205,10 @@ final class HealthCertificateReissuanceConsentViewModel {
 					Log.error("Reissuance request failed due to certificates being nil", log: .vaccination)
 				}
 				
+				let dispatchGroup = DispatchGroup()
```

Comment:
A comment would be nice to see why we need a DispatchGroup

Reply:
@marcussc  Done